src/main/java/decodex/modules/Base64Decoder.java

```diff
@@ -0,0 +1,30 @@
+package decodex.modules;
+
+import java.util.Base64;
+
+import decodex.data.Data;
+
+public class Base64Decoder extends BaseModule {
+
+    private final Base64.Decoder base64Decoder;
+
+    public Base64Decoder(String name, String description) {
```

Comment:
Module name and description is constant. I don't think the constructor needs to take in parameters. Similar with Base64Encoder.